lexicons/tools/ozone/moderation/queryEvents.json

```diff
@@ -106,6 +106,10 @@
               "type": "string"
             }
           },
+          "policy": {
+            "type": "string",
+            "description": "If specified, only events where the policy matches the given policy are returned"
+          },
```

Comment:
Is there a decent chance we end-up wanting to list multiple policies here in the future?  If so, we could proactively make this an array.